modules/labs-react/text-input/lib/TextInput.Field.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
```

Comment:
lol. I didn't even think to put a `.` in the file name. `TextInput.Field.tsx` instead of `TextInputField.tsx`. I'm not sure which I like better. The exported component variable is still `TextInputField`.

Reply:
I can drop for consistency with the export :D